index.js

```diff
@@ -55,10 +56,26 @@ function SRIHashAssets(inputNodes, options) {
 SRIHashAssets.prototype = Object.create(CachingWriter.prototype);
 SRIHashAssets.prototype.constructor = SRIHashAssets;
 
+SRIHashAssets.prototype.getBaseHREF = function getBaseHREF(string) {
+  var baseTag = string.match(BASE_CHECK);
+  if (baseTag && baseTag[0]) {
+    var href = baseTag[0].match(HREF_CHECK);
+    var relativePath = href && href[1];
+
+    if (!relativePath) { return null; }
+
+    return this.inputPaths[0] + relativePath;
```

Comment:
Hmm. I'm not sure? I haven't really seen a relative base url like that, is it valid?